.gitignore

```diff
@@ -0,0 +1,576 @@
+# Created by https://www.toptal.com/developers/gitignore/api/visualstudio,visualstudiocode,yarn,node,vercel
+# Edit at https://www.toptal.com/developers/gitignore?templates=visualstudio,visualstudiocode,yarn,node,vercel
+
+### Node ###
```

Comment:
When using a .gitignore file, you only need to specify the files or folders you want to ignore in your repository. It does not add files or folders that do not exist in your repository